EIPS/eip-2309.md

```diff
@@ -3,8 +3,7 @@ eip: 2309
 title: ERC-721 Consecutive Transfer Extension
 author: Sean Papanikolas (@pizzarob)
 discussions-to: https://github.com/ethereum/EIPs/issues/2309
-status: Last Call
-review-period-end: 2020-09-08
```

Comment:
Review period doesn't end for this until September (ISO-8601 date format).

Reply:
@MicahZoltu gah my mistake - it was supposed to be August 9 - two weeks from the start of last call status. Is there anything we can do here?

Reply:
Unfortunately, I think the best you can do is move back to DRAFT, then to Last Call again with a date that is 2 weeks from that date.  The reason for this is that it will trigger pushing the Last Call EIP to an RSS feed with a new correct date, and give people ample time to review before it is over.  This does mean that you'll have to wait at least 2 more weeks though.  😢

Reply:
If you decide to do this, ping me and I'll do my best to merge both (to Draft and then to Last Call) quickly.

Reply:
@MicahZoltu that's unfortunate. I will just leave as is. Would you recommend closing this PR and re-opening September 8?

Reply:
Either that or just mention me on September 8th.  :)

Reply:
@MicahZoltu today is the day! Can this be merged?